env.d/development/common

```diff
@@ -5,4 +5,5 @@ DJANGO_SECRET_KEY=ThisIsAnExampleKeyForDevPurposeOnly
 
 # Python
 PYTHONUNBUFFERED=1
```

Comment:
Out of scope here, but should be in the Dockerfile instead.

Reply:
Ok thanks, I moved it in the `development` image of the Dockerfile.

Reply:
It should also stand for production to ensure unbuffering of stderr/stdout, see: https://docs.python.org/3/using/cmdline.html#cmdoption-u

Reply:
done, thanks ! :+1: